spec/models/github_repository_spec.rb

```diff
@@ -150,6 +150,44 @@
       end
     end
 
+    context "importable" do
+      describe "#importing?", :vcr do
```

Comment:
You're testing the same thing in all 3 of these describes. I'd remove the unrelated expectations from each one. E.g. the #importing describe should only expect against the result of that method.